README.md

```diff
@@ -82,7 +88,13 @@ Write the BNF (not EBNF) description for this language.
 
 ## A2.1
 
-«replace this with your answer»
+<sentence> ::= "The"<adjective><noun><verb><adverb>"."
+
+<adjective> ::= <adjective> <adjective> 
+<adjective> ::= "smelly" | "lazy" | "" 
+<noun> ::= "dog" | "cat"
+<verb> ::= "ate" | "ran"
+<adverb> ::= "slowly" | "noisily" | ""
```

Comment:
# 7

This grammar only allows two adjectives, but I assume that's a typo in line 93.